pkgs/development/python-modules/torch/bin.nix

```diff
@@ -74,7 +93,9 @@ in buildPythonPackage {
     # Includes CUDA and Intel MKL, but redistributions of the binary are not limited.
     # https://docs.nvidia.com/cuda/eula/index.html
     # https://www.intel.com/content/www/us/en/developer/articles/license/onemkl-license-faq.html
-    license = licenses.bsd3;
+    # torch's license is BSD3.
+    # torch-bin includes CUDA and MKL binaries, therefore unfreeRedistributable is set.
+    license = licenses.unfreeRedistributable;
```

Comment:
We can update this to `[bsd3 issl unfreeRedistributable]` too

Reply:
Done